src/extension/importer/core/src/main/java/org/geoserver/importer/DataStoreFormat.java

```diff
@@ -191,7 +191,7 @@ public int getFeatureCount(ImportData data, ImportTask task) throws IOException
     public DataStore createDataStore(ImportData data) throws IOException {
         DataStoreFactorySpi dataStoreFactory = factory();
 
-        Map<String,Serializable> params = createConnectionParameters(data);
+        Map<String,Serializable> params = createConnectionParameters(data, null);
```

Comment:
createDataStore(ImportData data) seems to only ever be used for lightweight tasks such as verifying format, listing layers, and indirect import. Data stores created using this method are not added to the catalog, (in part because this seems to mostly be used where the catalog is not available), so we are safe using an absolute path here.